tests/unit_tests/test_features.py

```diff
@@ -76,3 +76,31 @@ def test_select_variation_variation(test_feature_variants):
     selected_variant = test_feature_variants.get_variant({"userId": "2"})
     assert selected_variant["enabled"]
     assert selected_variant["name"] == "VarB"
+
+
+def test_variant_metrics_are_reset(test_feature_variants):
+    test_feature_variants.get_variant({"userId": "2"})
+    assert test_feature_variants.variant_counts["VarB"] == 1
+
+    test_feature_variants.reset_stats()
+    assert test_feature_variants.variant_counts == {}
```

Comment:
Empty dicts are truthy so this should work. I'm more used to seeing this kind of thing in Python but if you prefer the comparison to empty dict for clarity that's fine too!

```suggestion
    assert not test_feature_variants.variant_counts
```